index.d.ts

```diff
@@ -0,0 +1,33 @@
+declare var Viz: VizJs.Viz
+export default Viz
+
+declare namespace VizJs {
+
+    interface Viz {
+        (src: string, opts?: VizOpts): string | HTMLImageElement
+        svgXmlToPngImageElement(svgXml: string, scale: number | undefined, callback: ImageCallback): void
+        svgXmlToPngImageElement(svgXml: string, scale?: number): HTMLImageElement
+        svgXmlToPngBase64(svgXml: string, scale: number | undefined, callback: ImageCallback): void
+    }
+
+    interface ImageCallback {
+        (error: Error | null, image: HTMLImageElement): void
+    }
+
+    type Format = "svg" | "xdot" | "plain" | "ps" | "json" | "png-image-element"
+    type Engine = "circo" | "dot" | "neato" | "osage" | "twopi"
+
+    interface VizOpts {
+        format?: Format
+        engine?: Engine
+        scale?: number
+        images?: ImageDimensions[],
```

Comment:
1.8.0 added a `files` option (which I forgot to add to the README) that takes an array of objects: `{ path: string, data: string }`.